tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Should be good to go with .tsx files now.  I would start migrating whatever is being worked on away from proptypes and toward typescript (starting where there arent complex types)